libraries/AP_Relay/AP_Relay.cpp

```diff
@@ -408,6 +408,10 @@ void AP_Relay::set_pin_by_instance(uint8_t instance, bool value)
 
     const bool initial_value = get_pin(pin);
 
+    if (_params[instance].invert) {
```

Comment:
```suggestion
    if (_params[instance].invert > 0) {
```
We quite commonly also allow -1 for disabled although its not documented.